server/src/instant/model/schema.clj

```diff
@@ -135,20 +141,63 @@
                                entities)]
     {:refs refs-indexed :blobs blobs-indexed}))
 
+(defn dup-message [etype label]
+  (str "Duplicate entry found for attribute: "
+       etype
+       "->"
+       label
+       ". "
+       "Check your schema file for duplicate link definitions."))
+
+(defn assert-unique-idents! [current-attrs steps]
+  (let [current-ident-names (->> current-attrs
+                                 (filter (comp #{:ref} :value-type))
```

Comment:
q: is it necessary to scope only to `ref` attrs?

Reply:
Good catch.  Duplicate blobs should be filtered out by `schemas->ops`, but we can make `assert-unique-idents!` more robust by removing this filter.